lib/jekyll-redirect-from/redirect.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en-US">
+  <meta charset="utf-8">
+  <title>Redirecting...</title>
```

Comment:
We mean to use an ellipsis. We can use an ellipsis. We should use an ellipsis.

#93